.travis.yml

```diff
@@ -1,21 +1,32 @@
 language: python
+cache: pip
+dist: xenial
 sudo: false
+
+services:
+  - postgresql
+  - mysql
+
+python:
+  - pypy
```

Comment:
pypy still declared in travis configuration, maybe we would want to remove this or maybe add to allowed_failures to track pypy support progresses

Reply:
This is more an issue with mysql/postgresql dependencies in pypy than the code itself, so i'm agree with @DragoonAethis who want remove it.

Reply:
Missed that one when searching for pypy files, will remove